assets/src/edit-story/app/media/pagination/test/reducer.js

```diff
@@ -183,4 +185,62 @@ describe('reducer', () => {
       expect.objectContaining({ pageToken: 'page2', nextPageToken: 'page3' })
     );
   });
+
+  it('should reset {next}pageToken when the search term changes', () => {
```

Comment:
shouldn't these tests be in media/media3p/test/providerReducer.js

Reply:
I believe this comment is outdated